platforms/src/GtcStaking/Providers/GtcStaking.ts

```diff
@@ -108,9 +143,17 @@ export class GtcStakingProvider implements Provider {
           }
         }, new BigNumber(0));
 
+        // SelfStake => total
+        const selfStakeV2: BigNumber = selfStakesV2.reduce((totalStake, currentStake) => {
+          if (new Date(currentStake.unlock_time) > new Date()) {
+            return totalStake.plus(new BigNumber(currentStake.amount));
+          }
+        }, new BigNumber(0));
```

Comment:
Does this work if the selfStakesV2 includes an expired stake in the middle of the list? I would think we'd need to do something like
```
const selfStakeV2: BigNumber = selfStakesV2.reduce((totalStake, currentStake) => {
  if (new Date(currentStake.unlock_time) > new Date()) {
    return totalStake.plus(new BigNumber(currentStake.amount));
   }
   return totalStake;
}, new BigNumber(0));
```
Otherwise "totalStake" will be set to undefined when the condition evaluates to false.

The one above should be fixed too (line 140).